web/server/entrypoint_arm64.go

```diff
@@ -0,0 +1,27 @@
+//go:build cgo
+
+package server
+
+import (
+	"go.viam.com/rdk/gostream"
+	"go.viam.com/rdk/gostream/codec/h264"
+	"go.viam.com/rdk/gostream/codec/opus"
+	"go.viam.com/rdk/gostream/codec/x264"
+	"go.viam.com/rdk/utils"
+	"strings"
+)
+
+var streamConfig gostream.StreamConfig
```

Comment:
Nothing outside this file uses this variable, so we don't need a lock.

Reply:
So we only ever call it once. It reads and parses a file, so I wouldn't want to add that overhead to `makeStreamConfig`.

Reply:
Right, but makeStreamConfig is only ever called once anyway AFAIK, so I don't think there's any savings there. And keeping it in a function (like every other entrypoint does) saves keeping a variable on the stack for the entire lifetime of the process. Granted, that's tiny and trivial, but I do think it's cleaner not to have init() "magic" where it can be avoided, and a package-level variable means other code anywhere in this package that might use the variable name "streamConfig" is easier to break. E.g. by accidentally using this variable when they shouldn't, because auto-complete in an IDE just filled it in.

Reply:
Good point. This has been updated.